spec/v2/index.bs

```diff
@@ -2237,7 +2238,7 @@ date: Mon, 23 May 2022 19:33:16 GMT
 content-type: application/json
 content-length: 1831
 server: Pathfinder
-link: &lt;https://api.pathfinder.sine.dev/2/footprints?limit=10&offset=10&gt;; rel="next"
+link: &lt;https://api.pathfinder.sine.dev/2/footprints?limit=10&amp;offset=10&gt;; rel="next"
```

Comment:
I think this change can be removed from this PR as well.

Reply:
It would not build the html without this change on my local machine (which is weird, given that it builds it in github actions...)